lib/workers/repository/update/branch/index.ts

```diff
@@ -553,26 +553,29 @@ export async function processBranch(
     await setStability(config);
     await setConfidence(config);
 
-    // break if we pushed a new commit because status check are pretty sure pending but maybe not reported yet
-    // but do not break when there are artifact errors
-    if (
+    const skipCondition =
+      commitSha &&
       !config.artifactErrors?.length &&
       !userRebaseRequested &&
```

Comment:
those should stay at original condition, as we should skip automerge on artifact errors and user rebase.

Reply:
the message in the above comment indicates the opposite 
`// but do not break when there are artifact errors` 
which means we do the Try auto merge flow

Reply:
what you say makes more sense though,